CONTRIBUTING.md

```diff
@@ -1,79 +1,101 @@
+
+We try to discuss all questions directly related to a feature or bug in the respective issue, in order to preserve it for the future and for other developers. We use discord for real-time communication.
+
+This is how we solve bugs and implement features, step by step:
+1. We find an issue we want to work on, usually during the sprint planning but as an open source contributor this can happen at any time.
+2. We communicate with the team to see if the issue is still available. (When you comment on an issue but don't get an answer there within 1-2 days try to mention @Human-Connection/hc-dev-team to make sure we check in.)
+3. We make sure we understand the issue in detail – what problem is it solving and how should it be implemented?
+4. We assign ourselves to the issue and move it to `In Progress` on [Zenhub](https://app.zenhub.com/workspaces/human-connection-nitro-5c0154ecc699f60fc92cf11f).
+5. We start working on it in a `new branch` and open a `pull request` prefixed with `[WIP]` (work in progress) to which we regularly push our changes.
+6. When questions come up we clarify them with the team (directly in the issue on Github).
+7. When we are happy with our work and our PR is passing all tests we remove the `[WIP]` from the PR description and ask for reviews (if you're not sure who to ask there is @Human-Connection/hc-dev-team which pings all core developers).
+8. We then incorporate the suggestions from the reviews into our work and once it has been approved it can be merged into master!
+
+Every pull request needs to:
+* fix an issue (if there is something you want to work on but there is no issue for it, create one first and discuss it with the team)
+* include tests for the code that is added or changed
+* pass all tests (linter, backend, frontend, end-to-end)
+* be approved by at least 1 developer who is not the owner of the PR (when more than 10 files were changed it needs 2 approvals)
+
+## The Team
+
+There are many volunteers all around the world helping us build this network and without their contributions we wouldn't be where we are today. Big thank you to all of you!
+
+You can see the core team behind Human Connection [on our website](https://human-connection.org/en/the-team/). On Github you will mostly run into our developers:
+* Robert (@roschaefer)
+* Matt (@mattwr18)
+* Wolle (@Tirokk)
+* Alex (@ogerly)
+* Alina (@alina-beck)
+* Martin (@datenbrei), our head of IT
+* and sometimes Dennis (@DennisHack), the founder of Human Connection
+
+## Meetings and Pair Programming Sessions
+
+Times below refer to **German Time** – that's CET (GMT+1) in winter and CEST (GMT+2) in summer – because most Human Connection core team members are living in Germany.
+
+Daily standup
+* every Monday–Friday 11:30
```

Comment:
> [<img alt="roschaefer" height="40" width="40" align="left" src="https://avatars1.githubusercontent.com/u/38261864?s=88&v=4">](https://github.com/roschaefer) **Authored by [roschaefer](https://github.com/roschaefer)**
_<time datetime="2019-11-12T15:46:06Z" title="Tuesday, November 12th 2019, 4:46:06 pm +01:00">Nov 12, 2019</time>_

---
> **Outdated (history rewrite)** - original diff
---
```diff
@@ -1,79 +1,101 @@
 # CONTRIBUTING
 
-Thanks so much for thinking of contributing to the Human Connection project, we really appreciate it! :-\)
+Thank you so much for thinking of contributing to the Human Connection project! It's awesome you're here, we really appreciate it. :-\)
 
 ## Getting Set Up
 
-Instructions for how to install all the necessary software can be found in our [documentation](https://docs.human-connection.org/human-connection/).
+Instructions for how to install all the necessary software and some code guidelines can be found in our [documentation](https://docs.human-connection.org/human-connection/).
 
-We recommend that new folks should ideally work together with an existing developer. Please join our [discord](https://discord.gg/6ub73U3) instance to chat with developers or just ask them in tickets in [Zenhub](https://app.zenhub.com/workspaces/human-connection-nitro-5c0154ecc699f60fc92cf11f/boards?repos=152252353):
+To get you started we recommend that you join forces with a regular contributor. Please join [our discord instance](https://human-connection.org/discord) to chat with developers or just get in touch directly on an issue on either [Github](https://github.com/Human-Connection/Human-Connection/issues) or [Zenhub](https://app.zenhub.com/workspaces/human-connection-nitro-5c0154ecc699f60fc92cf11f/boards?repos=152252353):
 
 ![](https://dl.dropbox.com/s/vbmcihkduy9dhko/Screenshot%202019-01-03%2015.50.11.png?dl=0)
 
-Here are some general notes on our development flow:
-
-## Development
-
-* Currently operating in two week sprints
-* We are using ZenHub to coordinate
-  * estimating time per issue is the crucial feature of [Zenhub](https://app.zenhub.com/workspaces/human-connection-nitro-5c0154ecc699f60fc92cf11f) that Github does not have
-  * "up-for-grabs" links to [Github project](https://github.com/Human-Connection/Human-Connection/issues?q=is%3Aopen+is%3Aissue+label%3A%22good+first+issue%22)
-  * ordering on ZenHub not necessarily reflected on github projects
-* AgileVentures run open pairing sessions at 10:30am UTC each week on Tuesdays and Thursdays
-* Core team
-  * all the people who are hired by HC non-profit corporation
-  * you can Meet-the-team [every two weeks in German](https://human-connection.org/veranstaltungen/) and [every month in English](https://human-connection.org/en/events/).
-  * 9 people
-    * 2 core developers \(Robert [@roschaefer](https://github.com/roschaefer) and Greg [@appinteractive](https://github.com/appinteractive)\)
-    * 3 marketeers Jasi, Dennis and Sensi
-    * Hardy doing business development
-    * Martin head of IT and previously data protection officer
-    * Victor doing accounting and controlling
-    * Nicolas is the community manager \(reviews content in the network\) reflects community opinion back to the core team
-* when can folks pair with Robert
-  * 10am UTC until 5pm UTC every working day
+We also have regular pair programming sessions that you are very welcome to join! We feel this is often the best way to get to know both the project and the team. Most developers are also available for spontaneous sessions if the times listed below don't work for you – just ping us on discord.
+
+## Development Flow
+
+We operate in two week sprints that are planned, estimated and prioritised on [Zenhub](https://app.zenhub.com/workspaces/human-connection-nitro-5c0154ecc699f60fc92cf11f). All issues are also linked to and synced with [Github](https://github.com/Human-Connection/Human-Connection/issues). Look for the `good first issue` label if you're not sure where to start!
+
+We try to discuss all questions directly related to a feature or bug in the respective issue, in order to preserve it for the future and for other developers. We use discord for real-time communication.
+
+This is how we solve bugs and implement features, step by step:
+1. We find an issue we want to work on, usually during the sprint planning but as an open source contributor this can happen at any time.
+2. We communicate with the team to see if the issue is still available. (When you comment on an issue but don't get an answer there within 1-2 days try to mention @Human-Connection/hc-dev-team to make sure we check in.)
+3. We make sure we understand the issue in detail – what problem is it solving and how should it be implemented?
+4. We assign ourselves to the issue and move it to `In Progress` on [Zenhub](https://app.zenhub.com/workspaces/human-connection-nitro-5c0154ecc699f60fc92cf11f).
+5. We start working on it in a `new branch` and open a `pull request` prefixed with `[WIP]` (work in progress) to which we regularly push our changes.
+6. When questions come up we clarify them with the team (directly in the issue on Github).
+7. When we are happy with our work and our PR is passing all tests we remove the `[WIP]` from the PR description and ask for reviews (if you're not sure who to ask there is @Human-Connection/hc-dev-team which pings all core developers).
+8. We then incorporate the suggestions from the reviews into our work and once it has been approved it can be merged into master!
+
+Every pull request needs to:
+* fix an issue (if there is something you want to work on but there is no issue for it, create one first and discuss it with the team)
+* include tests for the code that is added or changed
+* pass all tests (linter, backend, frontend, end-to-end)
+* be approved by at least 1 developer who is not the owner of the PR (when more than 10 files were changed it needs 2 approvals)
+
+## The Team
+
+There are many volunteers all around the world helping us build this network and without their contributions we wouldn't be where we are today. Big thank you to all of you!
+
+You can see the core team behind Human Connection [on our website](https://human-connection.org/en/the-team/). On Github you will mostly run into our developers:
+* Robert (@roschaefer)
+* Matt (@mattwr18)
+* Wolle (@Tirokk)
+* Alex (@ogerly)
+* Alina (@alina-beck)
+* Martin (@datenbrei), our head of IT
+* and sometimes Dennis (@DennisHack), the founder of Human Connection
+
+## Meetings and Pair Programming Sessions
+
+Times below refer to **German Time** – that's CET (GMT+1) in winter and CEST (GMT+2) in summer – because most Human Connection core team members are living in Germany.
+
+Daily standup
+* every Monday–Friday 11:45
```


```suggestion
* every Monday–Friday 11:30
```

Reply:
> [<img alt="alina-beck" height="40" width="40" align="left" src="https://avatars1.githubusercontent.com/u/38261864?s=88&v=4">](https://github.com/alina-beck) **Authored by [alina-beck](https://github.com/alina-beck)**
_<time datetime="2019-11-12T16:33:22Z" title="Tuesday, November 12th 2019, 5:33:22 pm +01:00">Nov 12, 2019</time>_

---


nooo, daily is at 11:45 – or am I just always 15 minutes late? 😜

Reply:
> [<img alt="mattwr18" height="40" width="40" align="left" src="https://avatars1.githubusercontent.com/u/38261864?s=88&v=4">](https://github.com/mattwr18) **Authored by [mattwr18](https://github.com/mattwr18)**
_<time datetime="2019-11-12T20:14:46Z" title="Tuesday, November 12th 2019, 9:14:46 pm +01:00">Nov 12, 2019</time>_

---


we agreed to change it since we are always rushing to get to the 12pm meeting @alina-beck

Reply:
> [<img alt="alina-beck" height="40" width="40" align="left" src="https://avatars1.githubusercontent.com/u/38261864?s=88&v=4">](https://github.com/alina-beck) **Authored by [alina-beck](https://github.com/alina-beck)**
_<time datetime="2019-11-13T05:28:56Z" title="Wednesday, November 13th 2019, 6:28:56 am +01:00">Nov 13, 2019</time>_

---


oh, I had no idea! I'll merge this now, thanks for updating :)

or no, I can't – somebody please approve?